CMakeLists.txt

```diff
@@ -213,7 +213,7 @@ if(BACKEND_CRASHPAD)
 		"$<BUILD_INTERFACE:${PROJECT_SOURCE_DIR}/external/crashpad/third_party/mini_chromium/mini_chromium>"
 	)
 	# FIXME: this should be applied to mini_chromium (or at least to crashpad_client/crashpad_handler)
-	target_compile_features(sentry PUBLIC cxx_std_11)
+	target_compile_features(sentry PUBLIC cxx_std_14)
```

Comment:
I wrote that fixme.
I added it because to fix warnings/errors of c++ standard in mini_chromium.
Since it is mini_chromium that needs c++14, it is that target that should have the compile feature added.
So this line should be added to the git submodule containing mini_chromium.
I did add the fixme here because I wanted to keep my cmake pr self containing.